tests/test_webhook_verification.py

```diff
@@ -10,7 +10,7 @@
     '{"resource_type":"check","action":"check.completed","object":'
     '{"id":"check-123",'
     '"status":"complete","completed_at_iso8601":"2020-01-01T00:00:00Z",'
-    '"href":"https://api.onfido.com/v3/checks/check-123"}}}'
+    '"href":"https://api.onfido.com/v3.1/checks/check-123"}}}'
```

Comment:
@Phoebe-B I realise in hindsight this should have v3 in the href, since it's the webhook version, not the API version